### PR TITLE
Add runbook for PrometheusRemoteWrite alerts

### PR DESCRIPTION
**Pull request by DebakelOrakel**
## Checklist

- [ ] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

File: docs/modules/ROOT/pages/runbooks/remotewrite.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+= Alert Group: syn-Prometheus
+
+== Alert Rule: SYN_PrometheusRemoteWriteBehind [[SYN_PrometheusRemoteWriteBehind]]
+== Alert Rule: SYN_PrometheusRemoteWriteDesiredShards [[SYN_PrometheusRemoteWriteDesiredShards]]
+
+=== icon:glasses[] Overview
+
+This alert indicates that the Mimir instance on APPUiO Cloud LPG-2 has a problem accepting metrics.
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/remotewrite.adoc:8]:
```suggestion
This alert may indicate that the remote write receiver isn't accepting metrics due to an internal problem or that there's a network issue between Prometheus and the remote write receiver.
```

File: docs/modules/ROOT/pages/runbooks/remotewrite.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Typically after maintenance, the nginx pod in front of Mimir, didn't renew the ip of the distributor pods.
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/remotewrite.adoc:9]:
```suggestion
If the remote write receiver is a Mimir instance, the root cause may be that the ngnix in front of the Mimir components has stale pod IPs in its DNS cache.
```

File: docs/modules/ROOT/pages/runbooks/remotewrite.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+=== icon:wrench[] Resolve
+
+Restart the nginx pod.
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/remotewrite.adoc:13]:
```suggestion
If you've identified that the Mimir nginx is the cause of the issue, restart the nginx pod.
```

File: docs/modules/ROOT/pages/runbooks/remotewrite.adoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
++
+[source,shell]
+----
+k -n vshn-appuio-mimir delete po -l app.kubernetes.io/component=nginx
+----
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/remotewrite.adoc:18]: Too VSHN-specific for component-openshift4-monitoring, we'll have to figure out a better place to keep runbooks with this much VSHN-specific information (maybe kb.vshn.ch/oc4) and overwrite the runbook_url in the global defaults.

```suggestion
```

File: docs/modules/ROOT/pages/runbooks/remotewrite.adoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
+= Alert Group: syn-Prometheus
+
+== Alert Rule: SYN_PrometheusRemoteWriteBehind [[SYN_PrometheusRemoteWriteBehind]]
+== Alert Rule: SYN_PrometheusRemoteWriteDesiredShards [[SYN_PrometheusRemoteWriteDesiredShards]]
+
+=== icon:glasses[] Overview
+
+This alert indicates that the Mimir instance on APPUiO Cloud LPG-2 has a problem accepting metrics.
+Typically after maintenance, the nginx pod in front of Mimir, didn't renew the ip of the distributor pods.
+
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/remotewrite.adoc:10]:
```suggestion

=== icon:[search] Investigate

* Check that the remote write receiver's endpoint is reachable
+
[source,bash]
----
kubectl -n openshift-monitoring --as=cluster-admin exec -it prometheus-k8s-0 -- curl <remote-write-endpoint>
----
* Check the remote write receiver for any issues.
If the remote write receiver is a Mimir instance, check the Mimir nginx pod logs.
Errors like the following indicate that nginx's DNS cache contains stale pod IPs.
+
[source]
----
2022/12/13 08:50:31 [error] 9#9: *2748893 vshn-appuio-mimir-distributor-headless.vshn-appuio-mimir.svc.cluster.local could not be resolved (110: Operation timed out), client: 10.128.10.35, server: , request: "POST /api/v1/push HTTP/1.1", host: "metrics-receive.appuio.net"
----
```

**Review by simu — CHANGES_REQUESTED** [review]
Overall, this runbook (without my suggested changes) is much too VSHN-specifc to be suitable for inclusion in the component. If we want a very VSHN-specific runbook, we should probably place it in kb.vshn.ch/oc4/ instead, and overwrite `runbook_url` in the VSHN global defaults.

Please remember that components should generally be kept in a form that they could be useful for someone else wanting to manage OpenShift 4 with Project Syn.

**Comment by DebakelOrakel** [comment]
> Overall, this runbook (without my suggested changes) is much too VSHN-specifc to be suitable for inclusion in the component. If we want a very VSHN-specific runbook, we should probably place it in kb.vshn.ch/oc4/ instead, and overwrite `runbook_url` in the VSHN global defaults.
> 
> Please remember that components should generally be kept in a form that they could be useful for someone else wanting to manage OpenShift 4 with Project Syn.

Yes that is true.

**Review by simu — APPROVED** [review]

**Pull request merged**